metadata.json

```diff
@@ -19,37 +19,56 @@
   ],
   "operatingsystem_support": [
     {
-      "operatingsystem": "RedHat"
+      "operatingsystem": "RedHat",
+      "operatingsystemrelease": [
+        "7",
+        "8",
+        "9"
+      ]
     },
     {
-      "operatingsystem": "CentOS"
+      "operatingsystem": "CentOS",
+      "operatingsystemrelease": [
+        "7",
+        "8"
+      ]
     },
     {
-      "operatingsystem": "OracleLinux"
+      "operatingsystem": "Scientific",
+      "operatingsystemrelease": [
+        "7"
+      ]
     },
     {
-      "operatingsystem": "Scientific"
+      "operatingsystem": "Debian",
+      "operatingsystemrelease": [
+        "10",
+        "11"
+      ]
     },
     {
-      "operatingsystem": "Debian"
+      "operatingsystem": "Ubuntu",
+      "operatingsystemrelease": [
+        "18.04",
+        "20.04",
+        "22.04"
+      ]
     },
     {
-      "operatingsystem": "Ubuntu"
-    },
-    {
-      "operatingsystem": "AIX"
-    },
-    {
-      "operatingsystem": "Archlinux"
+      "operatingsystem": "AIX",
+      "operatingsystemrelease": [
+        "7.1",
+        "7.2"
+      ]
     }
   ],
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 4.6.1 < 7.0.0"
+      "version_requirement": ">= 7.0.0 < 9.0.0"
```

Comment:
same here, I think dropping multiple major versions should be a dedicated PR

Reply:
Agreed, same as above.

Reply:
Created a PR for this https://github.com/puppetlabs/puppetlabs-lvm/pull/284